resources/kcp/charts/kyma-environment-broker/values.yaml

```diff
@@ -266,7 +266,7 @@ environmentsCleanup:
   imageName: "kyma-environments-cleanup-job"
   cronJobName: "environments-cleanup"
   maxAge: "24h"
-  version: "PR-2476"
+  version: "v20230321-8963fe01"
```

Comment:
Did you checked if this image is present in the registry?

Reply:
Yes, I took the version from the registry.